containers/Itunes/index.tsx

```diff
@@ -62,8 +63,8 @@ export const TracksContainer: React.FC<ItunesContainerProps> = ({ recommendation
     <>
       <Container>
         <StyledRecommnendationWrapper>
-          <T id="recommended" styles={fonts.style.subheading()} />
-          <Recommended recommendations={recommendations} />
+          <T id="recommended" />
+          <Recommended />
```

Comment:
But how are we passing the recommendations data to this component ?

Reply:
We aren't passing any props for now. It's just the UI for now.